homeassistant/components/group/light.py

```diff
@@ -132,19 +139,24 @@ class LightGroup(GroupEntity, LightEntity):
 
     _attr_available = False
     _attr_icon = "mdi:lightbulb-group"
-    _attr_is_on = False
+    _attr_is_on = False  # type: Optional[bool]
```

Comment:
`_attr_is_on` is already typed upstream

Reply:
I figured, but I couldn't find the declaration for it myself. The Binary Sensor Group uses `_attr_is_on = None` to represent a group with an unknown state, as I've also done here. Without setting it as an optional bool, [a mypy error is produced](https://github.com/home-assistant/core/runs/5626855067) stating that `_attr_is_on` must be a bool, which `None` ≠ `bool`. Is there a different way I should be going about this?

Reply:
According to mypy, [this line](https://github.com/home-assistant/core/blob/ce18637dcd427a300a6ba04e700a6df7b66dc49b/homeassistant/components/group/light.py#L142) was declaring `_attr_is_on` as a `bool`.

Reply:
I merged that change, tests seem to pass here locally on my system 👍